python/pyarrow/parquet.py

```diff
@@ -1272,9 +1464,11 @@ def _make_manifest(path_or_paths, fs, pathsep='/', metadata_nthreads=1,
 {1}
 filters : List[Tuple] or List[List[Tuple]] or None (default)
     List of filters to apply, like ``[[('x', '=', 0), ...], ...]``. This
-    implements partition-level (hive) filtering only, i.e., to prevent the
-    loading of some files of the dataset if `source` is a directory.
-    See the docstring of ParquetDataset for more details.
+    implements partition-level (hive) filtering, i.e., to prevent the
+    loading of some files of the dataset if `source` is a directory, as well
+    as file-level filtering (if `use_legacy_dataset` is set to False).
```

Comment:
Same as with the previous docstring.

Reply:
maybe extract `_filter_explanation_docstring` or similar?